.github/workflows/ts-packages.yml

```diff
@@ -60,45 +60,8 @@ jobs:
         env:
           CC_SECRET: ${{ secrets.CC_SECRET }}
 
-  # A hack that allows running a job only if a specific directory changed.
```

Comment:
I figured i could remove this entire block now that there's coverage across all ts-packages, but if there's a good reason to keep this, happy to add back in.